solutions/foregone-solution/go/foregone-solution.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	reader := bufio.NewReader(os.Stdin)
+	testCases, _ := reader.ReadString('\n')
```

Comment:
why are you ignoring the error?

Reply:
My bad. I just got the 3 checks in the code jam platform and overlooked this, i've just added an error handling function.
Thank you for your observation.